extensions.json

```
@@ -818,23 +818,22 @@
     {
       "id": "ms-vscode.js-debug",
       "repository": "https://github.com/microsoft/vscode-js-debug",
-      "version": "1.50.0",
-      "checkout": "v1.50.0",
-      "prepublish": "npm run compile && npm run package",
-      "extensionFile": "dist/js-debug.vsix"
+      "version": "1.50.1",
+      "checkout": "master",
```

The latest release tag seems to be `v1.50.2`:

```suggestion
      "checkout": "v1.50.2",
```

---

Done!